src/app/features/ledger/flows/jwt-signing/ledger-sign-jwt-container.tsx

```diff
@@ -106,6 +109,14 @@ export function LedgerSignJwtContainer() {
       return;
     }
 
+    // Low-grade code. This is to be removed when deprecating legacy APIs
```

Comment:
I'd tag the issue here so we can search when the time is right
```suggestion
    // TODO #4566 Low-grade code. This is to be removed when deprecating legacy APIs
```